modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -238,6 +238,14 @@ public static void Print(params object[] what)
             godot_icall_GD_print(GetPrintParams(what));
         }
 
+        /// <summary>
+        /// Prints a string including BBCode
+        /// </summary>
+        public static void Print_rich(params object[] what)
```

Comment:
The C# convention is to use PascalCase for method names.

```suggestion
        public static void PrintRich(params object[] what)
```

Reply:
Fixed this in my latest comment, thanks for letting me know!